movingTech.NY/Driver/StartRideScreen.java

```diff
@@ -48,6 +48,11 @@ public void startRide() throws InterruptedException {
         Thread.sleep(4000);
         driver.findElement(AppiumBy.xpath("//android.widget.TextView[@text='Start Ride']")).click();
         Thread.sleep(4000);
+    }
+	
+	public void tollRideStartRideScreen() {
+		
```

Comment:
@Aj-cmd-afk is there  any changes in the StartRideScreen or added extra spaces?

Reply:
It is just extra Space